src/plugins/mer/merhardwaredevicewizardpages.cpp

```diff
@@ -138,34 +139,40 @@ void MerHardwareDeviceWizardSelectionPage::handleTestConnectionClicked()
     sshParams.timeout = timeout();
     sshParams.authenticationType = SshConnectionParameters::AuthenticationTypeAll;
 
+    QString error;
```

Comment:
Nitpick, more often the name `errorMessage` is used.

Reply:
That's better than errorString suggested above. :wink: (QString errorString felt a bit redundant.)

Reply:
yeah I am a bit messing with these two here and there, but since now I will remember I am sure :)